autoload/leetcode.vim

```diff
@@ -15,6 +15,11 @@ if plugin_dir not in sys.path:
 if thirdparty_dir not in sys.path:
   sys.path.append(thirdparty_dir)
 
+if vim.eval('g:leetcode_china'):
```

Comment:
`vim.eval` returns a string for an integer, so the return value here is actually `"0"`, and hence always true.

Reply:
IMO one way to solve it is to use `int()` to convert it to an integer.